app/assets/stylesheets/application.scss

```diff
@@ -22,36 +22,156 @@
   width: 100%;
 }
 
-.body {
-  padding: 15px;
+#entry {
+  width: 100%;
+
+  .card-body form{
```

Comment:
I like how you targeted these classes by nesting them within the entry id

Reply:
CSS is a mystery to me still! A run-through on this would also be nice.